app/models/health/eligibility_response.rb

```diff
@@ -78,12 +78,18 @@ def TRN(subscriber)
     end
 
     def eligible(subscriber)
-      EB(subscriber).detect{ |eb| eb.first == '1' }
+      ebs = EB(subscriber)
+      masshealth = ebs.detect{ |eb| eb.first == '1' } || false
```

Comment:
detect will return the value it finds, did you mean `any?` (Same on the next line).  The end result is probably the same, but you aren't going to get a `true`.